docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -198,43 +198,192 @@ This table can then be queried in Trino::
-TEXT              VARCHAR
-TIMESTAMP         TIMESTAMP(3) WITH TIME ZONE
-TIMEUUID          UUID
-TINYINT           TINYINT
-TUPLE             ROW with anonymous fields
-UUID              UUID
-UDT               ROW with field names
-VARCHAR           VARCHAR
-VARINT            VARCHAR
-================  ======
-
-Any collection (LIST/MAP/SET) can be designated as FROZEN, and the value is
-mapped to VARCHAR. Additionally, blobs have the limitation that they cannot be empty.
-
-Types not mentioned in the table above are not supported.
+.. _cassandra-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and Cassandra each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Cassandra type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Cassandra types to the corresponding Trino types according to
+the following table:
+
+.. list-table:: Cassandra type to Trino type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Cassandra database type
+    - Trino type
+    - Notes
+  * - ``ASCII``
+    - ``VARCHAR``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    - Blobs have the limitation that they cannot be empty.
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``DECIMAL``
+    - ``DOUBLE``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``INET``
+    - ``IPADDRESS``
+    -
+  * - ``INT``
+    - ``INTEGER``
+    -
+  * - ``LIST<?>``
+    - ``VARCHAR``
+    - Any collection (``LIST``, ``MAP``, or ``SET``) can be designated as
+      ``FROZEN``.
+  * - ``MAP<?, ?>``
+    - ``VARCHAR``
+    -
+  * - ``SET<?>``
+    - ``VARCHAR``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``TEXT``
+    - ``VARCHAR``
+    -
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP(3) WITH TIME ZONE``
+    -
+  * - ``TIMEUUID``
+    - ``UUID``
+    -
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``TUPLE``
+    - ``ROW`` with anonymous fields
+    -
+  * - ``UDT``
+    - ``ROW`` with field names
+    -
+  * - ``UUID``
+    - ``UUID``
+    -
+  * - ``VARCHAR``
+    - ``VARCHAR``
+    -
+  * - ``VARINT``
+    - ``VARCHAR``
+    -
+
+No other types are supported.
+
+Trino type to Cassandra type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Trino types to the corresponding Cassandra types according to
+the following table:
+
+.. list-table:: Trino type to Cassandra type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Trino type
+    - Cassandra database type
+    - Notes
+
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``DOUBLE``
+    - ``DECIMAL``
```

Comment:
Remove. Trino double type is mapped to Cassandra double.

Reply:
Done